src/dndbeyond/content-scripts/character.js

```diff
@@ -873,24 +873,79 @@ function displayAction(paneClass) {
     });
 }
 
+function handleCustomText(paneClass) {
+    // Function Objective: Search for and handle block of VTT Message text...
+    //...in pane Notes or Description
+    const outCustomRollFuncs = {
+        before: [],
+        replace:[],
+        after:  [],
+        roll: (customTextFuncList, normalRollFunction=undefined) => {
+            if (customTextFuncList.length > 0) {
+                for (let i = 0; i < customTextFuncList.length; i++) {
+                    customTextFuncList[i]();
+                }
+            } else if (normalRollFunction) {
+                normalRollFunction();
+            }
+        },
+    };
+    const regexp = /```\[([^\]]*)\]\s*((\S|\s)*?)\s*```/g;
+    const rollOrderTypes = ["before", "after", "replace"];//(relative to normal roll msg)
+    const paneText = descriptionToString($(`.${paneClass}`).find(".ddbc-property-list__property:contains('Notes:'), .ct-action-detail__description, .ct-spell-detail__description, .ct-item-detail__description, .ddbc-action-detail__description, .ddbc-spell-detail__description, .ddbc-item-detail__description"));
```

Comment:
Forgot to mention on this one, it won't work, because the `selector.html()` will return the html of the first element if multiple elements are matching, so the single query to find notes and description and passing it to `descriptionToString` will only return the Notes.